app/src/main/kotlin/com/wire/android/ui/home/messagecomposer/EnabledMessageComposer.kt

```diff
@@ -73,18 +76,20 @@ fun EnabledMessageComposer(
     val density = LocalDensity.current
     val navBarHeight = BottomNavigationBarHeight()
     val isImeVisible = WindowInsets.isImeVisible
-    val offsetY = WindowInsets.ime.getBottom(density)
+    val height = WindowInsets.ime.getBottom(density)
```

Comment:
question: what height is it? I think it was more clear with previous name `offsetY` because it's used to move this composable up, but correct me if I'm wrong